### DISCUSSION TITLE
Is 'Feedback' feature in Insiders? - It's not working for me!!

### DISCUSSION THREAD

**Discussion by saurbhc** [squidfunk/mkdocs-material#5026] [Q&A]
- [Was this page helpful?](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#was-this-page-helpful)
- [Added support for feedback widget (Was this page helpful?) in 'mkdocs-material-8.4.0'](https://github.com/squidfunk/mkdocs-material/releases/tag/8.4.0)

I'm not in Insiders program, but wondering if the feature feedback ('Was this page helpful?') is in Insiders or Publically available?
As this feature is not working for me. I followed the exact same instructions in this [page](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#was-this-page-helpful) and accepted the analytics cookie too, but no luck. Could you please guide me if i'm doing something wrong?

**Comment by squidfunk** [comment]
No, it's available in the community edition [since version 8.4.0](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#was-this-page-helpful). In order to help you, we need a [reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) to see exactly how you configured the feedback widget. It's likely a misconfiguration. Please create one and attach it here as a zip.

**Reply by saurbhc** [reply]
Here's the example.zip for your reference: [example.zip](https://github.com/squidfunk/mkdocs-material/files/10715964/example.zip)

**Reply by squidfunk** [reply]
Thanks for providing. You're missing the [`provider`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#google-analytics) setting:

<img width="713" alt="Bildschirmfoto 2023-02-12 um 13 18 51" src="https://user-images.githubusercontent.com/932156/218310620-09a1cd34-e0c4-42fe-bd7d-bc7d66a8f312.png">

**Reply by Marlony-Gabriel** [reply]
Hello people. I'm note sure if this is right place for my question. If not, I  apologize in advance.
I'm using analytics with feedback widget, as explained here: https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/

This is my mkdocs.yml part that configure booth:

[mkdocs-analitics.txt](https://github.com/squidfunk/mkdocs-material/files/11092669/mkdocs-analitics.txt)

And I followed all the steps described in the section to view the data collected.

Go to your Google Analytics dashboard

Go to the configure page on the left hand menu, then select custom definitions

Click the custom metrics tab and then create custom metrics, enter the following values:

Metric name: Page helpful
Description: Was this page helpful?
Event parameter: data
Unit of measurement: Standard
Go to the explore page on the left hand menu, create a new blank exploration

Configure the report as follows:

Dimensions: Add Event name and Page location
Metrics: Add Event count and Page helpful (the custom metric created in step 3)
Rows: Page location
Values: Drag in both Event count and Page helpful
Filters: Add a new filter for Event name / exactly matches / feedback

Now, when i go to Analytics and select the  created report, I'm getting this:

![1](https://user-images.githubusercontent.com/129212251/228326953-811fabd8-f215-4628-95c3-8cf4cfffff7e.png)

Not this

![2](https://user-images.githubusercontent.com/129212251/228327025-6cc830b7-0f09-4a78-b1bb-19ec62915afc.png)

I'm asking here because maybe i'm missing some configuration on report or mkdocs. If not,  google just had changed the way these reports are making.  If it's the second case, means that i'm need  to figure out how to do the  same report with this new  configurations on google  and nothing is wrong with my mkdocs page.
